ENT-8817 - Non-reentrant ConcurrentHashMap.compute() causes deadlock - added release notes entry (#881)
corda/corda-docs-portal: content/en/platform/corda/4.10/community/release-notes.md

* Previously, Archive Service commands did not write messages to the log files unless an error or issue occurred. An update now means that messages are also written when commands are run successfully. For more information, refer to [Archive Service Command-Line Interface (CLI)](..\..\..\..\tools\archiving-service\archiving-cli.md)

* The opentelemetry tracing signal is now supported in flows across nodes.

> * A rare condition was found when database transactions are rolled back under heavy load that caused flow state machine threads to stop processing flows, leading to eventual node lockup in certain circumstances.  This fix prevents this from happening.

Reviewer: ```suggestion
* A rare condition was found when database transactions were rolled back under heavy load that caused flow state machine threads to stop processing flows. This resulted in eventual node lockup in certain circumstances.  This fix prevents this from happening.
```

Author: Done.
